docs/source/introduction/quickstart.ipynb

```diff
@@ -22,45 +22,56 @@
     "\n",
```

Comment:
<ul><li>about its value</li></ul><p>When talking about the marginal standard deviation, we should mention that this corresponds to the classic error estimate. The covariance additionally relates entries of the solution to each other.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>added.</p>